tests/layer_tests/tensorflow_tests/test_tf_BatchMatrixInverse.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestBatchMatrixInverse(CommonTFLayerTest):
+    def prepare_input(self, inputs_info):
+        assert 'x:0' in inputs_info, "Test error: inputs_info must contain `x`"
+        x_shape = inputs_info['x:0']
+        inputs_data = {}
+        x = np.random.rand(*x_shape).astype(np.float32)
+        for idx in np.ndindex(x_shape[:-2]):
+            x[idx] += np.eye(x_shape[-1]) * x_shape[-1]
+        inputs_data['x:0'] = x
+        return inputs_data
+
+    def create_batch_matrix_inverse_net(self, input_shape, data_type):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(data_type, input_shape, 'x')
+            y = tf.linalg.inv(x)
```

Comment:
please use `tf.raw_ops.BatchMatrixInverse` instead of `inv` to make sure that you are really testing this operation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/BatchMatrixInverse.